:memo: Clarify generic CDN setup (#2867)
platformsh/platformsh-docs: docs/src/domains/cdn/cloudflare.md

To properly configure your Cloudflare CDN,
see the Cloudflare official documentation on [how to get started](https://developers.cloudflare.com/cache/get-started/).

> Make sure you have a [custom domain](../steps/_index.md) set up and that your CDN points to your [project target](../../domains/steps/_index.md#2-get-the-target-for-your-project)
> for the [DNS challenge to succeed](../troubleshoot.md#ownership-verification).

Reviewer: :hammer_and_wrench: This is trying to get two ideas into one sentence, which seems less than ideal. At least, I don't think the custom domain part relates to the DNS challenge.

```suggestion
Then set up a [custom domain](../steps/_index.md).
To get the [DNS challenge to succeed](../troubleshoot.md#ownership-verification),
have your CDN point to your [project's target URL](../../domains/steps/_index.md#2-get-the-target-for-your-project).
```

Author: The custom domain part is required so that the domain is recognized as being live (plan size > dev + custom domain = live), so that deactivates the temporary internal domain on prod (although the pointing to the gw is still required - through the `edge_hostname`).

I think it's still related to the DNS setup, since you have to configure both your DNS and the CDN properly to make sure that the ownership verification goes through successfully.

Maybe "DNS challenge" is a poor choice of words on my end for this part. It's checked _through_ the DNS, but yeah.
